plugins/foreman_openscap/1.0/index.md

```diff
@@ -0,0 +1,630 @@
+
+Detailed variable description can be found in the role's [documentation](https://galaxy.ansible.com/theforeman/foreman_scap_client).
+
+### 2.3.3 Manual
+
+Manual policy deployment does not require to install anything, but leaves configuration to user. See section [4.6.2](plugins/foreman_openscap/0.9/index.html#4.6.2Manualpolicydeployments) for details.
+
+# 3. Upgrading from 0.4.x
+
+As with any Foreman plugin, the recommended upgrade path is done via `yum upgrade`. This will ensure that all of the packages are with the right version, dependencies
+and data migrations.
+
+__Note: ARF reports are not automatically migrated and need a manual step__
+
+## 3.1 Migrating ARF reports from 0.4.x
+
+ARF reports in 0.5.x and Foreman > 1.11 are now part of Foreman's reports (Reports STI), and the physical ARF report XML is now saved at the Smart-Proxy.
+
+This requires a special migration, which needs both Foreman and Smart-Proxy up and running, with latest respective OpenSCAP plugins installed.
+
+Since we cannot assure this during upgrade, migration of ARF reports has moved to a rake task which should be performed after upgrading.
+
+To upgrade ARF reports from 0.4.x to 0.5.x:
+
+* Upgrade foreman_openscap to version 0.5.x
+* Upgrade smart_proxy_openscap to version 0.5.x
+* Run `foreman-rake db:migrate` to ensure all other data has been migrated
+* Restart Foreman and Smart-Proxy
+* Run `foreman-rake foreman_openscap:migrate[<proxy_id>]` (Where `<proxy_id>` is the id of the Smart-Proxy with OpenSCAP feature).
+
+__Notes:__ to find out your Smart-Proxy id you can either run `hammer proxy list` or pick the Smart-Proxy's id from the url in web UI. please note it should be inside the square brackets
+
+__Process:__ During the ARF reports migration, the old ARF reports are fetched from the Foreman database, sent to the Smart-Proxy for re-processing and saving and are sent back to the Foreman in their new
+format - while keeping their original data.
+
+Once the old ARF report has successfully migrated, it is deleted from the old table.
+
+# 4. Usage
+
+This chapter covers features that you can use in terms of Foreman and OpenSCAP
+integration. Everything described below assumes you've sucessfully completed the installation steps described in previous sections.
+
+*__Please note:__ smart_proxy_openscap is __required__ for the normal operation of foreman_openscap*
+
+You would usually start with uploading SCAP contents, then create policies of those SCAP contents and assign the policy to hosts or hostgroups.
+
+## 4.1 Creating SCAP content
+
+### 4.1.1 Creating default SCAP content
+
+When installing foreman_openscap from RPM, we also add default SCAP content provided by [scap-security-guide](https://fedorahosted.org/scap-security-guide/).
+
+In previous versions, the default SCAP content was added via seed task.
+
+In version >= 0.5.x, we are processing all of OpenSCAP content and reports in the Proxy.
+
```

Comment:
we are on version 1.x, so it doesn't matter what was used before, just what is needed now.